spec/spec_helper.rb

```diff
@@ -6,6 +6,8 @@
 require 'rspec'
 require 'rspec/autorun' # somehow makes rcov work with rspec
 require 'twitter_cldr'
+require 'coveralls'
+Coveralls.wear!
```

Comment:
According to the docs, `#wear!` causes results to be sent to Coveralls whenever the project is built on Travis. I did a cursory read of the source code and it didn't look like that was true... I must have missed something.

Also, twitter-cldr-rb uses Simplecov already. We might consider hooking up Coveralls' Simplecov integration.